app/utils/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+export const errorHandler = (err, req, res, next) => {
+  console.error(err.stack);
+
+  res.status(err.status || 500).json({
+    success: false,
+    message: err.message || "Internal Server Error",
+    stack: err.stack,
```

Comment:
user shouldn't know about stack
this information only for developers

Reply:
Thank you, I'll rework it with next commit